src/pages/UploadApk/UploadApk.js

```diff
@@ -1,21 +1,155 @@
+                                {i18n.t('Create app version')}
+                            </h2>
+
+                            <ReactFinalForm.Form onSubmit={handleSubmit}>
+                                {({ handleSubmit, valid, submitting }) => (
+                                    <form onSubmit={handleSubmit}>
+                                        <ReactFinalForm.Field
+                                            required
+                                            name="version"
+                                            label={i18n.t('Version')}
+                                            placeholder={i18n.t('e.g. 1.0.0')}
+                                            component={InputFieldFF}
+                                            className={styles.field}
+                                            validate={composeValidators(
+                                                hasValue,
+                                                createPattern(
+                                                    versionRegex,
+                                                    versionValidationMessage
+                                                )
+                                            )}
+                                        />
+                                        <ReactFinalForm.Field
+                                            required
+                                            filterable
+                                            name="androidOSVersion.min"
+                                            label={i18n.t(
+                                                'Minimum Android version'
+                                            )}
+                                            placeholder={i18n.t(
+                                                'Select a version'
+                                            )}
+                                            component={SingleSelectFieldFF}
+                                            className={styles.field}
+                                            validate={hasValue}
+                                            options={androidOSVersion}
+                                        />
+                                        <ReactFinalForm.Field
+                                            filterable
+                                            name="androidOSVersion.recommended"
+                                            label={i18n.t(
+                                                'Recommended Android version'
+                                            )}
+                                            placeholder={i18n.t(
+                                                'Select a version'
+                                            )}
+                                            component={SingleSelectFieldFF}
+                                            className={styles.field}
+                                            options={androidOSVersion}
```

Comment:
Hey, 
As far as I understand the recommended version should be equal to or higher than the minimum version. Does it make sense to only show equal or higher options in this dropdown?
Currently, it is possible to create an APK with a recommended version smaller than the minimum version. Is this configuration useful to the users or is it a bug?

<img width="891" alt="Screenshot 2023-04-05 at 17 06 43" src="https://user-images.githubusercontent.com/6817073/230124128-a25f42dc-1203-46d0-bf54-d1911b498b3d.png">

Thanks!